package.json

```diff
@@ -20,13 +20,12 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": [
-      "react-app",
-      "react-app/jest"
-    ]
+    "eject": "react-scripts eject",
+    "lint": "eslint . --fix",
+    "lint:check": "eslint .",
+    "format": "prettier . --write",
+    "format:check": "prettier . --check",
+    "prepare": "husky install"
```

Comment:
nitpick: it might be nice to have a rule that lints and formats in the same check, to make it easier for a developer to do everything at once. Maybe a rule like `all` and `all:check`?

Reply:
Implemented in 01766f0